vxdetector/Output_counter.py

```diff
@@ -18,58 +22,120 @@ def directory_navi(file_name, path, dir_name, dir_path):
     return file_name, dir_name, dir_path
 
 
-def create_output(path, file_name, unaligned_count, most_probable_V,
-                  probability, dir_name, dir_path):
+def rawincount(filename):
+    f = open(filename, 'rb')
+    bufgen = takewhile(lambda x: x, (f.raw.read(1024*1024)
+                                     for _ in repeat(None)))
+
+    return sum(buf.count(b'\n') for buf in bufgen)
+
+
+def region_count(dictionary, unaligned_count, temp_path, all_reads, mode):
```

Comment:
it might be worth switching to `pandas` instead of operating on dictionaries: https://pandas.pydata.org/